composer.json

```diff
@@ -124,7 +120,8 @@
         "drupal/media_expire": "^1.0",
         "drupal/nexx_integration": "^1.0||^2.0",
         "drupal/metatag": "^1.0",
-        "drupal/paragraphs": "1.1",
+        "drupal/paragraphs": "^1.0",
+        "drupal/paragraphs_features": "dev-1.x",
```

Comment:
we need to wait for paragraphs features release, and change this line before merge